lib/transform.js

```diff
@@ -205,7 +205,7 @@ function transform(source, template, options = {}) {
   if (elementId) {
     attrs.push(b.attr('id', b.text(elementId)));
   }
-  attributeBindings.forEach((value, key) => {
+  attributeBindings.forEach((key, value) => {
```

Comment:
hmm, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/forEach says that the order is `value, key` 🤔

Reply:
but generally I think you're correct that this is reversed. we should however fix this in the function that analyzes the `attributeBindings`.

Reply:
yeap, you're 💯 right...
sorry, jumped the gun a bit on the fix....updated, let me know what you think